types/mongodb/index.d.ts

```diff
@@ -2433,9 +2433,9 @@ export interface GridFSBucketOptions {
     readPreference?: ReadPreferenceOrMode;
 }
 
-/** http://mongodb.github.io/node-mongodb-native/3.1/api/GridFSBucket.html#~errorCallback */
-export interface GridFSBucketErrorCallback {
-    (err?: MongoError): void;
+/** http://mongodb.github.io/node-mongodb-native/3.6/api/GridFSBucket.html#~errorCallback */
+export interface GridFSBucketErrorCallback<T = any> extends MongoCallback<T> {
+    (error: MongoError, result: T): void;
 }
```

Comment:
Oh, I just saw the `extends` part.

You don't need to declare the function interface in this body, it will be inherited from `MongoCallback`.

Since this is called `GridFSBucketErrorCallback` I think that it should not take a `T` parameter.

```suggestion
export interface GridFSBucketErrorCallback extends MongoCallback<void> {}
```